models/topics.model.js

```diff
@@ -16,4 +16,16 @@ function findArticleById(articleId){
     })
 }
 
-module.exports = {findALlTopics, findArticleById}
\ No newline at end of file
+function findAllArticles(){
+    // const commentCount = db.query(`SELECT COUNT(body) AS comment_count FROM comments WHERE `)
+    return db.query(`SELECT articles.*, COUNT(comments.comment_id) AS comment_count
```

Comment:
We actually dont want to be returning the body from articles here so articles.* is not correct here unfortunately.